[2023-12-19T17:31:02.820+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: producer_dag.produce_task scheduled__2023-12-19T17:30:00+00:00 [queued]>
[2023-12-19T17:31:02.826+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: producer_dag.produce_task scheduled__2023-12-19T17:30:00+00:00 [queued]>
[2023-12-19T17:31:02.826+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-12-19T17:31:02.836+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): produce_task> on 2023-12-19 17:30:00+00:00
[2023-12-19T17:31:02.840+0000] {standard_task_runner.py:57} INFO - Started process 889 to run task
[2023-12-19T17:31:02.842+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'producer_dag', 'produce_task', 'scheduled__2023-12-19T17:30:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/producer.py', '--cfg-path', '/tmp/tmpf5ikjbx0']
[2023-12-19T17:31:02.844+0000] {standard_task_runner.py:85} INFO - Job 17: Subtask produce_task
[2023-12-19T17:31:02.875+0000] {task_command.py:410} INFO - Running <TaskInstance: producer_dag.produce_task scheduled__2023-12-19T17:30:00+00:00 [running]> on host b045abe315e4
[2023-12-19T17:31:02.921+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='producer_dag' AIRFLOW_CTX_TASK_ID='produce_task' AIRFLOW_CTX_EXECUTION_DATE='2023-12-19T17:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-19T17:30:00+00:00'
[2023-12-20T02:24:27.030+0000] {taskinstance.py:1097} INFO - Dependencies not met for <TaskInstance: producer_dag.produce_task scheduled__2023-12-19T17:30:00+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2023-12-20T02:24:27.031+0000] {taskinstance.py:1097} INFO - Dependencies not met for <TaskInstance: producer_dag.produce_task scheduled__2023-12-19T17:30:00+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2023-12-20T02:24:27.034+0000] {local_task_job_runner.py:154} INFO - Task is not able to be run
